.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "subprojects/hashmap.c"]
+	path = subprojects/hashmap.c
+	url = https://github.com/engelmi/hashmap.c.git
```

Comment:
I am fine with such approach for now but before merging this commit I would open an issue to track and get it resolved in upstream (get the original repo a meson file) or an official nack from the maintainer so we keep the fork. Sounds a good plan?

Reply:
As I wrote in https://github.com/containers/bluechi/pull/432#discussion_r128721298, I opened an issue in the original repo https://github.com/tidwall/hashmap.c/issues/32 and wait for a response. As soon as this PR is merged, I can create an issue for us to keep track of it.